application/frontend/web/index.php

```diff
@@ -1,16 +1,5 @@
 <?php
-defined('YII_DEBUG') or define('YII_DEBUG', true);
-defined('YII_ENV') or define('YII_ENV', 'dev');
 
-require(__DIR__ . '/../../vendor/autoload.php');
-require(__DIR__ . '/../../vendor/yiisoft/yii2/Yii.php');
-require(__DIR__ . '/../../common/config/bootstrap.php');
-require(__DIR__ . '/../config/bootstrap.php');
+$application = require('load-app.php');
```

Comment:
Moving it to a different folder would solve the problem of it being web-accessible, but I think I would lean towards simply duplicating the relevant pieces, because the tests are inherently separate from the application itself, so it seems fine that they each have a copy of this, especially since they need to be different. I hit a similar situation in SilAuth of needing to initialize a Yii application but not `run()` it. Thoughts?